s3/src/main/java/ch/cyberduck/core/s3/S3WebIdentityTokenExpiredResponseInterceptor.java

```diff
@@ -0,0 +1,67 @@
+package ch.cyberduck.core.s3;
```

Comment:
Verify if implementation can be shared with `S3TokenExpiredResponseInterceptor`.